### DISCUSSION TITLE
openTofu

### DISCUSSION THREAD

**Discussion by noriban** [bpg/terraform-provider-proxmox#603] [Q&A] [answered]
Is this provider will be available in opentofu?

**Comment by bpg-dev** [comment, marked as answer]
Yes, it will be! The efforts are covered in https://github.com/bpg/terraform-provider-proxmox/issues/562, feel free to comment on that ticket.

**Reply by noriban** [reply]
Thank you for reply :)

**Reply by loganmancuso** [reply]
ive been using it with my opentofu and its been working great!
